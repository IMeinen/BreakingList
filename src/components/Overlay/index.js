import React from 'react';
import PropTypes from 'prop-types';
import { OverlayContainer,Content } from './styles';

export default function Overlay({children}) {

  return (
    <OverlayContainer test-id="overlay-container">
      <Content test-id="overlay-content">{children}</Content>
    </OverlayContainer>
  );
}

Overlay.propTypes = {
  children: PropTypes.node.isRequired,
}
