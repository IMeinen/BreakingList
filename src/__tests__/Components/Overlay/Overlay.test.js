import React from 'react';

import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Overlay from '../../../components/Overlay';

Enzyme.configure({ adapter: new Adapter() });
describe('Overlay component tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Overlay>
        <div />
      </Overlay>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
