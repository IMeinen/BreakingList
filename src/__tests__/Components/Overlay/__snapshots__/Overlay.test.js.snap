// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay component tests renders correctly 1`] = `
<styled.div
  test-id="overlay-container"
>
  <styled.div
    test-id="overlay-content"
  >
    <div />
  </styled.div>
</styled.div>
`;
